script.js

```diff
@@ -2,7 +2,62 @@
 
 jQuery(function(){
     jQuery('ul.plugin_slider').bxSlider({
```

Comment:
can't you simply call `.bxSlider(JSINFO['plugin_slider'])` instead of repeating each setting?